app/controllers/StripeController.scala

```diff
@@ -193,7 +193,9 @@ class StripeController(paymentServices: PaymentServices, stripeConfig: Config, c
       }
 
       withParsedStripeHook(request.body) { stripeHook =>
-        val stripeService = paymentServices.stripeServices(stripeHook.mode)
+        val countryGroup: Option[CountryGroup] = CountryGroup.byFastlyCountryCode(stripeHook.fastlyCountryCode)
```

Comment:
Just to check, do we only want to switch to AUS on the country code ? Should we check for currency in AUD as well ? Can people pay in AUD from other countries ??

Reply:
@dominickendrick the only way to pay in AUD is by choosing that country group, and we'll use the right key in that case. The reason I'm inferring it from the header here is because this comes from the Stripe hook where the country group is stored in the metadata.

Reply:
Ah ok, makes sense : )